hack/update-codegen.sh

```diff
@@ -18,44 +18,43 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# We need to explicitly pass GO111MODULE=off to k8s.io/code-generator as it is significantly slower otherwise,
-# see https://github.com/kubernetes/code-generator/issues/100.
-export GO111MODULE=off
```

Comment:
Why is this removed?

Reply:
It didn't feel so slow without, but after measuring it is 44s vs 21s on my machine.
Added it again. Thanks for the hint.